vm-setup/roles/firewall/defaults/main.yml

```diff
@@ -1,13 +1,17 @@
 use_firewalld: true
 baremetal_interface: baremetal
 provisioning_interface: provisioning
+cluster_provisioning_interface: ironicendpoint
```

Comment:
`CLUSTER_PROVISIONING_INTERFACE` with default value of `ironicendpoint` is defined in `vars.md`, we should not hardcode it here?

Reply:
I just remembered why we put this here, it's note exported, not sure if it's a bug, didn't have the time to investigate, but without this here ansible  will complain

Reply:
I did not mean to remove it, but take it from env with a fallback, like the other variables around it. Like:
`cluster_provisioning_interface: "{{ lookup('env', 'CLUSTER_PROVISIONING_INTERFACE') | default('ironicendpoint', true) }}" `

Reply:
oh yeah, my bad, will change that